cli/src/commands/extensions/mod.rs

```diff
@@ -1,12 +1,17 @@
+mod api;
 mod extension;
 
-use std::{collections::HashSet, convert::TryFrom, fs, io::ErrorKind, path::PathBuf};
+pub(crate) use api::api_decls;
+pub use extension::*;
```

Comment:
Honestly not a big fan of `pub use` in general. I feel like things are going to be a ton more obvious if we just make the modules public?